cluster/examples/kubernetes/edgefs/cluster.yaml

```diff
@@ -55,7 +55,7 @@ subjects:
   name: rook-edgefs-cluster
   namespace: rook-edgefs
 ---  
-apiVersion: apps/v1
+apiVersion: policy/v1beta1
```

Comment:
This fixes #2681, right? If this needs backport to v0.9, then the change needs to be in a different commit, as the other changes are introducing a new feature.

Reply:
no need to backport